### DISCUSSION TITLE
parameterization__texturing_from_registered_rasters XCB error

### DISCUSSION THREAD

**Discussion by JAzorinL** [cnr-isti-vclab/PyMeshLab#74] [Q&A]
Hi all,

When I'm trying to calculate the texture from a set of camera rasters using  

`ms.apply_filter('parameterization__texturing_from_registered_rasters')`

I have the following error:

> [xcb] Unknown request in queue while appending request
> [xcb] Most likely this is a multi-threaded client and XInitThreads has not been called
> [xcb] Aborting, sorry about that.
> python: ../../src/xcb_io.c:151: append_pending_request: La declaración `!xcb_xlib_unknown_req_pending' no se cumple.

I'm not sure what is happening and could be related to GL... Any idea to fix it?

Thank you very much in advance.

**Comment by alemuntoni** [comment]
Sorry, I missed this thread.
I solved recently an issue about arameterization__texturing_from_registered_rasters (#76). 
However, this seems to be a different issue. What kind of environment are using to run your script? Is it some kind of server?
The filter requires an OpenGL context, therefore GL should be available in the running machine.
